### ISSUE TITLE
react-intl-universal-extract Error: Cannot find module './extract'

### ISSUE THREAD

**Issue by Hexi1997** [alibaba/react-intl-universal#205]
version: ^1.0.4

code 
```
const intlTool = require('react-intl-universal-extract');
console.log(intlTool)
```

nodejs version: 18.12.0

**Comment by Hexi1997** [comment]
error detail
```
Error: Cannot find module './extract'
Require stack:
- /home/hexi/Learn/react-intl-universal-demo/node_modules/react-intl-universal-extract/lib/index.js
- /home/hexi/Learn/react-intl-universal-demo/intl-extract.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
    at Module._load (node:internal/modules/cjs/loader:841:27)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/home/hexi/Learn/react-intl-universal-demo/node_modules/react-intl-universal-extract/lib/index.js:9:16)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/hexi/Learn/react-intl-universal-demo/node_modules/react-intl-universal-extract/lib/index.js',
    '/home/hexi/Learn/react-intl-universal-demo/intl-extract.js'
  ]
}
```

**Comment by cwtuan** [comment]
Thanks for reporting this issue. 
It's fixed in `react-intl-universal-extract@1.0.6`
